apps/src/templates/sectionsRefresh/SectionsSetUpContainer.jsx

```diff
@@ -70,17 +79,23 @@ const saveSection = (e, section) => {
 
   const csrfToken = document.querySelector('meta[name="csrf-token"]')
     .attributes['content'].value;
-  const loginType = queryParams('loginType');
-  const participantType = queryParams('participantType');
 
   const section_data = {
     login_type: loginType,
     participant_type: participantType,
+    course_offering_id: section.course.courseOfferingId,
+    course_version_id: section.course.versionId,
```

Comment:
just checking, do any of these or the ones in `recordSectionSetupEvent` need `?.` safe accessors?

Reply:
I don't think*** a section would exist without a course (even if the params in there are null) but I don't know if I have the time to totally verify that so I added them just to be safe!